core/src/main/java/tc/oc/pgm/stats/StatsMatchModule.java

```diff
@@ -93,6 +93,7 @@ public class StatsMatchModule implements MatchModule, Listener {
   private final Match match;
   private final Map<UUID, PlayerStats> allPlayerStats = new HashMap<>();
   private final Table<Team, UUID, PlayerStats> stats = HashBasedTable.create();
+  private final List<StatType.OfFormula> ofFormulas;
```

Comment:
"of formula" is not really the type of this variable, the variable is "formula stat type", just so happens that Formula is inner class to StatType hence it's written as StatType.Formula (and, since formula is already a type elsewhere and the collision would've been annoying to deal with, instead we opted for OfFormula) 

consider overall renaming variable names to formulaStats or similar
```suggestion
  private final List<StatType.OfFormula> formulaStats;
```

Reply:
Yeah definitely should rename